src/main/java/com/cloudbees/jenkins/support/api/FilePathContent.java

```diff
@@ -49,7 +53,25 @@ public void writeTo(OutputStream os) throws IOException {
         try {
             file.copyTo(os);
         } catch (InterruptedException e) {
-            throw new IOException2(e);
+            throw new IOException(e);
+        } catch (IOException e) {
+            if (isFileNotFound(e) || isFileNotFound(e.getCause())) {
+                OutputStreamWriter osw = new OutputStreamWriter(os, "utf-8");
+                try {
```

Comment:
`try-with-resources`?

Reply:
the `os` comes from the caller which is supposed to manage its lifecycle. We don't want to close it, even though there are provisions made to prevent `Content` classes from closing this stream.